english/explore-outside/talk-child/talk-more/nice-talk/nice-talk.md

```diff
@@ -1,3 +1,7 @@
 The child says, "Oh OK, so sorry for trapping you. Must have been a misunderstanding."
 
-"Here's $500 and the keys to my car."
\ No newline at end of file
+Suddenly, a large wave forms and capsizes the child's boat. What do you do?
+
+[Help the green child](./help-child/help-child.md)
+
+[Walk away](./ignore-child/ignore-child.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.